src/main/java/org/jabref/gui/WelcomePage.java

```diff
@@ -0,0 +1,209 @@
+package org.jabref.gui;
+
+import javafx.beans.binding.BooleanBinding;
+import javafx.event.EventHandler;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.control.SplitPane;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import org.jabref.gui.actions.ActionFactory;
+import org.jabref.gui.importer.ImportCommand;
+import org.jabref.gui.importer.NewDatabaseAction;
+import org.jabref.gui.shared.ConnectToSharedDatabaseCommand;
+import org.jabref.gui.util.TaskExecutor;
+import org.jabref.model.util.FileUpdateMonitor;
+import org.jabref.preferences.PreferencesService;
+
+public class WelcomePage {
+    String fontStyle = """
```

Comment:
Css belongs in base.css. please reuse the existing themed styling already in use instead of defining new font styles (except you really have to) and/or other styles.